### PR TITLE
Vaishnavi_Assignment_5_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image

### CODE REVIEW

File: src/vaishnavi_SeleniumBasics/MultiDropdown.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+			// Now we need to scroll down for accessing MultiDropdown for that we need JavascriptExecutor
+
+			JavascriptExecutor js = (JavascriptExecutor) driver;
+			js.executeScript("arguments[0].scrollIntoView();", driver.findElement(By.xpath("//select[2]")));
+
+			WebElement multiSelect = driver.findElement(By.xpath("//select[2]"));
+			Select ms = new Select(multiSelect);
+
+			// Getting count for total number of options
+			List<WebElement> li = ms.getOptions();
```
Comment (KrishnaKTechnocredits) [inline src/vaishnavi_SeleniumBasics/MultiDropdown.java:39]: Please give a proper variable name next time.

File: src/vaishnavi_SeleniumBasics/MultiDropdown.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			for (WebElement num : li) {
+				if (Integer.parseInt(num.getText()) % 2 == 0) {
+					num.click();
+					System.out.println(num.getText());
+				} 
+			}
+			
+			// Print All deselcted options
+			System.out.println("All the Deselected Option are: ===================");
+			for (WebElement num : li) {
```
Comment (KrishnaKTechnocredits) [inline src/vaishnavi_SeleniumBasics/MultiDropdown.java:55]: I hope you know that you can merge select and deselect logic, If not let me know.

File: src/vaishnavi_SeleniumBasics/MultiDropdown.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnavi_SeleniumBasics/MultiDropdown.java:39]: --Yes, Krishna I will remember from next time

File: src/vaishnavi_SeleniumBasics/MultiDropdown.java (hunk 2 above)
Comment (VaishnaviVJun20) [reply src/vaishnavi_SeleniumBasics/MultiDropdown.java:55]: -- Yes, I did that logic only before but then I wasn't happy with the way my output was looking so I added extra loop.
-- Please suggest If I should have gone with single logic ?

**Pull request merged**